### PR TITLE
Fix GELU ONNX export

### PR DESCRIPTION
**Pull request by nzmora-nvidia**
* Wrap GELU export with cast to/from FP32 to achieve same compute precision as TE.
* Increase GELU export test thresholds.
* Change export to ONNX opset 17 for smaller representation of LN (single node instead of subgraph).
* Remove the need for LN work-around for ORT

### CODE REVIEW

**Review by timmoon10 — COMMENTED** [review]
LGTM aside from the linting errors.

**Review by ksivaman — APPROVED** [review]

File: tests/pytorch/test_onnx_export.py [hunk 1]
```diff
@@ -311,7 +309,7 @@ def forward(self, inp):
 @pytest.mark.parametrize(
     "precision,     atol", [
     [torch.float32, 1e-7],
-    [torch.float16, 2e-3]
+    [torch.float16, 1e-7]
```
Comment (ksivaman) [inline tests/pytorch/test_onnx_export.py:312]: Looks like this is too strict, let's dial it back a bit

Comment (nzmora-nvidia) [reply tests/pytorch/test_onnx_export.py]: Fixed

**Pull request merged**